main_routers/cloudsave_router.py

```
@@ -0,0 +1,402 @@
+        )
+
+    local_exists = _local_character_exists(config_manager, name)
+    if local_exists:
+        released_memory_handle = await release_memory_server_character(
+            name,
+            reason=f"云存档下载前释放 SQLite 句柄: {name}",
+        )
+        if not released_memory_handle:
+            return _cloudsave_error_response(
+                "MEMORY_SERVER_RELEASE_FAILED",
+                "释放本地角色记忆句柄失败，已阻止下载覆盖，请稍后重试",
```

Ruff (0.15.9)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)